src/Components/RecsListTable/RecsListTable.spec.ct.js

```diff
@@ -482,3 +476,5 @@ describe('Recs list is requested with additional parameters №2', () => {
     getChipGroup('Clusters impacted').contains('.pf-c-chip', '1 or more');
   });
 });
+
+// TODO what will happen on request error?
```

Comment:
this
![image](https://user-images.githubusercontent.com/31385370/156578456-b6477a86-e947-42ea-8d5a-595a4ecf367f.png)

Reply:
Is there any way to test that? Maybe we should create a test for that. What do you think? Or is not needed?

Reply:
Just asking because if I am not mistaken we check the error status in other tables

Reply:
Yes, you can simply put `isError` to the `query` object within props. Also, the same message will appear when `isSuccess` is set to `true`, but `data` has no entries (recommendations). This cannot happen because, in the response, there will be at least the recommendations that do not hit any clusters.